cabfile/cabfile.go

```diff
@@ -132,9 +186,9 @@ func New(r io.ReadSeeker) (*Cabinet, error) {
 	if (hdr.Flags&hdrPrevCabinet) != 0 || (hdr.Flags&hdrNextCabinet) != 0 {
 		return nil, errors.New("multi-part Cabinet files are unsupported")
 	}
-	if (hdr.Flags & hdrReservePresent) != 0 {
-		return nil, errors.New("Cabinet files with reserved fields are unsupported")
-	}
+
+	// skip abReserve by skipping ahead cbCFHeader bytes
+	ioutil.ReadAll(io.LimitReader(r, int64(hdr.CBCFHeader)))
```

Comment:
I'd prefer if we wouldn't swallow the error here. And maybe there could be a meaningful error if we hit EOF as part of the seeking?

Reply:
I've changed this to `io.ReadFull` in 99593de99f00a0280258504ac5b938e7f4810dd5 and the error check is included now.

Reply:
I like the update you suggest

Reply:
Thank you!